src/core/zowe/core_for_zowe_sdk/logger.py

```diff
@@ -44,7 +44,7 @@ def registerLogger(name: str):
         return logger
 
     @staticmethod
-    def setLoggerLevel(level: int):
+    def setAllLoggerLevel(level: int):
```

Comment:
Should we add this renaming to the changelog (since it it's a prerelease breaking change)? 😋 
<img width="762" alt="image" src="https://github.com/user-attachments/assets/27f008d3-9646-40b0-b8b3-5d745bab878b">